technology/typemapping/store/jdbc/src/main/java/com/github/mizool/technology/typemapping/store/jdbc/AbstractJdbcValueLoadStrategy.java

```diff
@@ -0,0 +1,22 @@
+package com.github.mizool.technology.typemapping.store.jdbc;
+
+import java.util.List;
+
+import lombok.Getter;
+
+import com.github.mizool.technology.typemapping.business.DataType;
+
+abstract class AbstractJdbcValueLoadStrategy implements JdbcValueLoadStrategy
+{
+    @Getter
+    private final List<Integer> sourceColumnType;
+
+    @Getter
+    private final DataType targetDataType;
+
+    protected AbstractJdbcValueLoadStrategy(List<Integer> sourceColumnType, DataType targetDataType)
```

Comment:
Add a second constructor
```java
protected AbstractJdbcValueLoadStrategy(Integer sourceColumnType, DataType targetDataType)
{
    this(ImmutableSet.of(sourceColumnType), targetDataType);
}
```

Having one source column type seems to be the more common case, so this constructor allows to reduce boilerplate code.